mlflow/anthropic/autolog.py

```diff
@@ -0,0 +1,45 @@
+import inspect
+
+import mlflow
+import mlflow.anthropic
+from mlflow.entities import SpanType
+from mlflow.utils.autologging_utils.config import AutoLoggingConfig
+
+
+def _get_span_type(task_name: str) -> str:
+    span_type_mapping = {
+        "create": SpanType.CHAT_MODEL,
+    }
+    return span_type_mapping.get(task_name, SpanType.UNKNOWN)
```

Comment:
Do we need this function? Anthoropic does not have `task` concept and we only patch single `create` function.

Reply:
anthropic has two beta API that might be GA soon ("batch messages" and "count tokens"). 
I kept this function here in case we want to log "count tokens" API too, which should have type `SpanType.LLM`

Reply:
Makes sense! Could you add a short comment in the function?